opentracing-spring-cloud-starter-jaeger/README.md

```diff
@@ -0,0 +1,113 @@
+## Configuration
+
+```xml
+<dependency>
+  <groupId>io.opentracing.contrib</groupId>
+  <artifactId>opentracing-spring-cloud-starter-jaeger</artifactId>
+</dependency>
+```
+
+Simply adding the dependency to the application will ensure that all required OpenTracing and Jaeger dependencies are transitively resolved.
+Furthermore the dependency will ensure that Spring Boot will auto configure all the necessary OpenTracing beans when the application starts.  
+
+## Configuration options
+
+All the available configuration options can be seen in [JaegerConfigurationProperties](src/main/java/io/opentracing/contrib/spring/cloud/starter/jaeger/JaegerAutoConfiguration.java).
+The prefix to be used for these properties is `opentracing.jaeger`.
+Furthermore, the service name is configured via the standard Spring Cloud `spring.application.name` property.
+
+Beware to use the correct syntax for properties that are camel-case in `JaegerConfigurationProperties`.
+
+* For properties / yaml files use `-`. For example `opentracing.jaeger.log-spans=true`
+* For environment variables use `_`. For example `OPENTRACING_JAEGER_LOG_SPANS` 
+
+## Defaults
+
+If no configuration options are changed and the user does not manually provide any of the beans that the 
+auto-configuration process provides, the following defaults are used:
+
+* `unknown-spring-boot` Will be used as the service-name if no value has been specified to the property `spring.application.name`. 
+* `CompositeReporter` is provided which contains the following delegates:
+  - `LoggingReporter` for reporting spans to the console
+  - `RemoteReporter` that contains a `UdpSender` that sends spans to `localhost:6831` 
+* `ConstSampler` with the value of `true`. This means that every trace will be sampled
+* `NoopMetricsFactory` is used - effectively meaning that no metrics will be collected
+
+## Common cases
+
+### Set service name 
+
+Set `spring.application.name` to the desired name
+
+### HTTP Sender
```

Comment:
Could senders/reporters be in a one dedicated section?

Log Spans does not have to be there, we can abstract user of how the span is logged.